explainer.md

```diff
@@ -331,24 +331,53 @@ customElements.define("custom-tablist", CustomTabList);
+
+
+| **AT event**          | **Targets**                                                       | Orientation/Direction | **DOM event**                      |
+| --------------------- | ----------------------------------------------------------------- | --------------------- | ---------------------------------- |
+| `click` or `press`    | _all elements_                                                    |                       | `click`                            |
+| `focus`               | _all focusable elements_                                          |                       | `focus`                            |
+| `select`              | Elements whose computed role supports `aria-selected`             |                       | `click`                            |
+| `dismiss` or `escape` | _all elements_                                                    |                       | `Escape` KeyboardEvent             |
+| `contextMenu`         | _all elements_                                                    |                       | `contextmenu`                      |
+| `increment`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Up` KeyboardEvent                 |
+|                       | ""                                                                | horizontal LTR        | `Right` KeyboardEvent              |
+|                       | ""                                                                | horizontal RTL        | `Left` KeyboardEvent               |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Up` KeyboardEvent                 |
+| `decrement`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Down` KeyboardEvent               |
+|                       | ""                                                                | horizontal LTR        | `Left` KeyboardEvent               |
+|                       | ""                                                                | horizontal RTL        | `Right` KeyboardEvent              |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Down` KeyboardEvent               |
+| `scrollByPage`        | TBD (possibly custom scroll views)                                |                       | TBD (possibly `PageUp`/`PageDown`) |
+| `scrollIntoView`      | TBD                                                               |                       | No equivalent DOM event            |
+| `setValue`            | n/a                                                               |                       | No equivalent DOM event            |
+
+Notes on the previous table:
+- KeyboardEvent sequences include keyup/keydown/keypress.
```

Comment:
We discussed in the past having a brief delay between keydown and keyup to make it less obvious that the events are synthesized by the browser. Do we still like that idea? Should it be noted here?

Also note that `keypress` is deprecated and not fired in Chromium at all any more, so we would not be firing it for synthesized keyboard events.

Reply:
> We discussed in the past having a brief delay between keydown and keyup to make it less obvious that the events are synthesized by the browser. Do we still like that idea? Should it be noted here?

No. That's an implementation detail that UAs should just fix. And it's not specific to this (also mousedown/mouseup), so I don't think it needs to be mentioned here... 

> Also note that keypress is deprecated and not fired in Chromium at all any more, so we would not be firing it for synthesized keyboard events.

Good to know. I must've missed that spec change. Do you know of relevant links I should read about the context for the change? Click is convenient for authors since it only fires when nothing else cancels either mouseup or mousedown. Keypress was the same when nothing else cancelled either keyup or keydown... Looking into this. 👀

Reply:
> No. That's an implementation detail that UAs should just fix. And it's not specific to this (also mousedown/mouseup), so I don't think it needs to be mentioned here...

This is already a non-normative document... if we think UAs should (or shouldn't) do this, we may as well make a note?

https://w3c.github.io/uievents/#legacy-keyboardevent-event-types has some info on `keypress`, via https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event

Reply:
I'm not certain this is the location for something like that. Do you have suggested language?